src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int height = 2;
```

Comment:
hardcode. This field value should be received from user

Reply:
Okay, I got it